sympy/functions/special/error_functions.py

```diff
@@ -222,6 +222,18 @@ def _eval_as_leading_term(self, x):
         else:
             return self.func(arg)
 
+    def _eval_aseries(self, n, args0, x, logx):
+        from sympy.series.order import Order
+        point = args0[0]
+
+        if point is S.Infinity:
```

Comment:
What are the other possible values for ``point``?

Reply:
This is the only possible value of `point`.